pkgs/tools/security/wordlists/rockyou.nix

```
@@ -0,0 +1,37 @@
+{ fetchFromGitLab
+, lib
+, stdenv
+}:
+
+let
+  shared = "rockyou.txt";
+
+in stdenv.mkDerivation rec {
+  pname = "rockyou";
+  version = "0.3-1kali3";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "wordlists";
+    rev = "debian/${version}";
+    sha256 = "sha256-viOd8iKLTiHrPF0azICp/F16JrPUfo4kbcGUPD2/cRs=";
+  };
```

For congruence with the other descriptions, it might be a good idea to put `+ "rockyou.txt.gz"` after `}`:
```suggestion
  } + "rockyou.txt.gz";
```
Then, do the opposite of this change to the `gzip` unpacking line.